livelossplot/input_plugins/generic_keras.py

```diff
@@ -23,16 +25,12 @@ def loss2name(loss):
         return loss
 
 
-class _PlotLossesCallback():
-    def __init__(self, **kwargs):
+class _PlotLossesCallback:
+    def __init__(self, outputs=(matplotlib.Matplotlib,), **kwargs):
         self.liveplot = PlotLosses(**kwargs)
+        self.matplotlib_output = list(filter(lambda o: isinstance(o, matplotlib.Matplotlib), outputs))[0]
```

Comment:
This look like a dangerous pattern.

What if:

* there are a few matplitlib outputs (we don't guard against that; and in principle, it may make sense - some for display, other for saving)?

* there is not matplotlib instance?

Reply:
ok, I should have replaced it with list